api/app/views/mno_enterprise/jpi/v1/impac/alerts/_alert.json.jbuilder

```diff
@@ -1,7 +1,4 @@
 json.ignore_nil!
-json.extract! alert, :id, :title, :webhook, :service, :sent
+json.extract! alert, :id, :title, :webhook, :service, :sent, :recipients
 json.metadata alert.settings
 json.kpi_id alert.impac_kpi_id
-json.recipients alert.recipients.map do |recipient|
```

Comment:
should work because mnohub was already returning an array of hashes, is that correct?